xain/grpc/coordinator.py

```diff
@@ -103,112 +113,336 @@ def update_expires(self, participant_id):
             )
 
 
-def monitor_heartbeats(participants, terminate_event):
+class Coordinator:
+    """Class implementing the main Coordinator logic. It is implemented as a
+    state machine that reacts to received messages.
+
+    The states of the Coordinator are:
+        STANDBY: The coordinator is in standby mode, typically when waiting for
+        participants to connect. In this mode the only messages that the
+        coordinator can receive are :class:`~.coordinator_pb2.RendezvousRequest`
+        and :class:`~.coordinator_pb2.HeartbeatRequest`.
+
+        ROUND: A round is currently in progress. During a round the only
+        messages the coordinator can receive are
+        :class:`~.coordinator_pb2.StartTrainingRequest` and
+        :class:`~.coordinator_pb2.EndTrainingRequest`.
```

Comment:
Is this right? Aren't heartbeat requests still serviced? And rendezvous' can also be told to try `LATER`?

Reply:
That is true but I just wanted to make the documentation as simple as possible.

I left a note https://github.com/xainag/xain/pull/128/files#diff-6fad7f1e8d3e49838ad5b60a5cd57f52R143

Reply:
I would suggest rewording "...the **only** messages..." to "...the **important** messages..." for clarity.